### PR TITLE
pushing download link to details

### PR DESCRIPTION
**Pull request by lefherz**
the download links should go to the checks, the comments from the bot are too noisy.

### CODE REVIEW

File: .github/workflows/build-preview.yml [hunk 1]
```diff
@@ -71,6 +71,14 @@ jobs:
...
 #               Linux: https://download.delta.chat/desktop/preview/deltachat-desktop-${{ steps.prepare.outputs.prid }}.deb \
 #               Windows: https://download.delta.chat/desktop/preview/deltachat-desktop-${{ steps.prepare.outputs.prid }}.exe \
+    - name: "Add links to details"
+      if: steps.prepare.outputs.uploadtoserver
+      run: |
+        export GITHUB_API_URL="https://api.github.com/repos/deltachat/deltachat-desktop/statuses/${{ github.sha }}"
+        export STATE="success"
+        export ARTIFACT_URL="https://download.delta.chat/desktop/preview/deltachat-desktop-${{ steps.prepare.outputs.prid }}.AppImage"
+        export STATUS_DATA="{\"state\": \"${STATE}\", \"description\": \"The build was a success. Download it here:\", \"context\": \"Build Upload\", \"target_url\": \"${ARTIFACT_URL}\"}"
+        curl --fail --silent -X POST --user ":${{ GITHUB_TOKEN }}" "$GITHUB_API_URL" --data "$STATUS_DATA"
```
Comment (Simon-Laux) [inline .github/workflows/build-preview.yml]: `${{ GITHUB_TOKEN }}` this syntax is the right one here I believe, try $GITHUB_TOKEN

Comment (lefherz) [reply .github/workflows/build-preview.yml]: I think it used the correct syntax, the logs show the usual xxxx for secrets...

Comment (lefherz) [reply .github/workflows/build-preview.yml]: But the username is missing

**Comment by lefherz** [comment]
@Simon-Laux is this how you imagined it?

**Comment by r10s** [comment]
> the download links should go to the checks, the comments from the bot are too noisy.

yeah, i also thought about that, it was becoming harder to follow whats going on in this repo. thanks for fixing that :)

**Pull request merged**